nebula-logger/main/logger/classes/FlowLogRepo.cls

```diff
@@ -0,0 +1,18 @@
+/*************************************************************************************************
+* This file is part of the Nebula Logger project, released under the MIT License.                *
+* See LICENSE file or go to https://github.com/jongpie/NebulaLogger for full license details.    *
+*************************************************************************************************/
+
+public without sharing class FlowLogRepo {
```

Comment:
Would you mind also adding a dedicated test class (`FlowRepo_Tests`) for this class?